apps/backoffice-v2/src/domains/entities/hooks/useSelectNextEntity/useSelectNextEntity.tsx

```diff
@@ -1,14 +1,7 @@
-import { useSelectEntity } from '../useSelectEntity/useSelectEntity';
-import { useNextEntityIdQuery } from '../queries/useNextEntityIdQuery/useNextEntityIdQuery';
 import { useCallback } from 'react';
 
 export const useSelectNextEntity = () => {
-  const { data: nextId } = useNextEntityIdQuery({
-    initialState: {
-      sortBy: 'caseCreatedAt',
-    },
-  });
-  const onSelectNextEntity = useSelectEntity();
-
-  return useCallback(() => onSelectNextEntity(nextId)(), [onSelectNextEntity, nextId]);
+  return useCallback(() => {
```

Comment:
This hook is currently unused. I removed its implementation because it used queries that are deleted. We'll need to implement it once we'll return using it.